docs/content/direct/wds.md

```diff
@@ -2,68 +2,85 @@
 
-### Using the KubeStellar Core Helm Chart
+### Use the KubeStellar Core Helm Chart to create your WDS
 
 The recommended approach is to use the KubeStellar Core Chart:
 
 ```shell
 helm upgrade --install ks-core oci://ghcr.io/kubestellar/kubestellar/core-chart \
-  --set-json='WDSes=[{"name":"wds1", "type":"k8s"}]'
+  --set-json='WDSes=[{"name":"<wds-name>", "type":"<space-type>"}]'
 ```
 
-You can customize your WDS by specifying:
-- `name`: A unique name for the WDS
-- `type`: 
-  - `k8s` (default): Creates a basic Kubernetes API Server with a subset of kube controllers
-  - `host`: Uses the KubeFlex hosting cluster itself
-- `APIGroups`: A comma-separated list of API Groups to include
-- `ITSName`: The name of the ITS to be used by this WDS (required if multiple ITSes exist)
+Specify a `name` for your WDS, and you can customize it further with optional parameters:  
 
-### Using the KubeFlex CLI
+- `name` (required): A unique name for the WDS (`<wds-name>`)
+- `type` (optional, default: `k8s`):  (`<space-type>`)
 
-You can also create a WDS using the KubeFlex CLI:
+    - `k8s`: Creates a basic Kubernetes API Server with a subset of kube controllers
+    - `host`: Uses the KubeFlex hosting cluster itself. 
+    
+- `APIGroups` (optional): A comma-separated list of API Groups to include
+- `ITSName` (optional): The name of the ITS to be used by this WDS (required if multiple ITSes exist)
 
-```shell
-kflex create wds1 -p kubestellar
-```
+The type of space you choose determines the type of controllers that are included in your WDS. When creating a WDS, the recommended space types are "K8s" (the default space) and "host". If you the decide to use your hosting cluster's control plane (which is typically a KubeFlex cluster for Kubstellar) by specifying `type:host`, it presents you some benefits such as:
 
-This command creates a WDS and runs a post-create hook that deploys the KubeStellar controller manager and transport controller.
+- Saves the cost of creating an additional space.
+- Simplifies the architecture by reusing the hosting cluster
+- Makes the WDS directly accessible through the hosting cluster's API server
+
+> You can create multiple WDSes of the same space types and even a mix of different space types, where you can have the default and host WDSes in your cluster. A multiple WDS architecure can be valueable in cases where you want to manage workload runs across different sets of users or groups. Thus each user or group can use the same ITS for WEC inventory but with their different WDS
+
+### Creating a kubeconfig context for accessing the WDS
 
-## KubeFlex Hosting Cluster as WDS
+After creating your WDS, you will need to access it to submit workload objects. To do this you have to create a context for your WDS in kubeconfig context and set it context as your current cluster context:
```

Comment:
The current text has a problem with a misplaced modifier. The text "in kubeconfig" appears right after "your WDS" --- so the normal English interpretation would be that "in kubeconfig" is modifying "your WDS". But that is not what you mean. I would put "in your kubeconfig" _before_ "for your WDS" rather than after it.